activiti-cloud-services-audit-jpa/src/test/java/org/activiti/cloud/services/audit/jpa/converters/ErrorReceivedEventConverterTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.services.audit.jpa.converters;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.activiti.api.runtime.model.impl.BPMNErrorImpl;
+import org.activiti.api.runtime.model.impl.ProcessInstanceImpl;
+import org.activiti.cloud.api.process.model.events.CloudBPMNErrorReceivedEvent;
+import org.activiti.cloud.api.process.model.impl.events.CloudBPMNErrorReceivedEventImpl;
+import org.activiti.cloud.services.audit.jpa.events.ErrorReceivedAuditEventEntity;
+import org.junit.Test;
+
+public class ErrorReceivedEventConverterTest {
+
+    private ErrorReceivedEventConverter eventConverter = new ErrorReceivedEventConverter(new EventContextInfoAppender());
+
+    @Test
+    public void should_convert_toEntityErrorReceivedEvent() {
+        CloudBPMNErrorReceivedEventImpl event = createErrorReceivedEvent();
+
+        ErrorReceivedAuditEventEntity auditEventEntity = (ErrorReceivedAuditEventEntity) eventConverter.convertToEntity(event);
+
+        assertThatIsEqualTo(auditEventEntity, event);
+    }
+    
+    @Test
+    public void shoudld_convert_toAPIErrorReceivedEvent() {
```

Comment:
typo "should", and extra underscore between "convert" and "toAPIErrorReceivedEvent"

Reply:
Name is changed